pkg/polaris/cloudcluster/cloudcluster.go

```diff
@@ -230,19 +372,19 @@ func (a API) CreateCloudCluster(ctx context.Context, input cloudcluster.CreateAw
 		case gqlevent.ActivityStatusQueued:
 		case gqlevent.ActivityStatusRunning:
 		case gqlevent.ActivityStatusTaskSuccess:
-			a.log.Printf(log.Info, "AWS cloud cluster create in progress: %s\n", activitySeries.Activities.Nodes[0].Message)
+			a.log.Printf(log.Info, "cloud cluster create in progress: %s\n", activitySeries.Activities.Nodes[0].Message)
```

Comment:
It probably makes sense to check if there is at least 1 node and get the message for that node out at the top of the function. So the code doesn't panic if there is something wrong with the response from RSC.